src/eventInput/MouseEventInput.ts

```diff
@@ -23,6 +40,19 @@ export class MouseEventInput extends EventInput {
     return 0;
   }
 
+  protected _getButton(event: MouseEvent): string {
+    switch (event.buttons) {
+      case 1:
+        return "left";
+      case 2:
+        return "right";
+      case 4:
+        return "middle";
+      default:
+        return "left";
```

Comment:
Does this mean that the side buttons of the mouse are also considered as a left click? I think they shouldn't.